misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/reporting/translating/TranslationSupplierBasedTranslatorSpec.kt

```diff
@@ -46,27 +49,29 @@ abstract class TranslationSupplierBasedTranslatorSpec(
     }
 
     @Suppress("unused")
-    fun SpecBody.checkUsesDefaultOfTranslatable(testee: Translator) {
-        it("uses ${Translatable::class.simpleName}'s ${Translatable::getDefault.name}") {
-            val result = testee.translate(translatableHello)
-            expect(result).toBe(translatableHello.value)
+    fun ScopeBody.checkUsesDefaultOfTranslatable(testee: Translator) {
+        describe("checkUsesDefaultOfTranslatable") {
```

Comment:
no need for the describe, we are stating the same in the `it`

Reply:
According to spek2 rules, _it_ block should be wrapped inside a describe block

Reply:
That's true `it` is not even available otherwise. However checkUsesDefaultOfTranslatable is used within a describe and therefore no additional `describe` is necessary. Maybe you have to change the receiver `ScopeBody` in case `it` is not available.